projectile.el

```diff
@@ -211,6 +211,12 @@ Otherwise consider the current directory the project root."
   :group 'projectile
   :type 'string)
 
+(defcustom projectile-use-gtags nil
```

Comment:
I think we should simply give the users the option to explicitly select what they want to use - gtags, etags-select or find-tag. Boolean values are not a good idea when you have multiple options.